pkg/kubelet/kubelet.go

```diff
@@ -2028,6 +2028,9 @@ func (kl *Kubelet) HandlePodAdditions(pods []*v1.Pod) {
 		if !kl.podIsTerminated(pod) {
 			// Only go through the admission process if the pod is not
 			// terminated.
+			
+			// Get all pod in the pod manager.
+			existingPods := kl.podManager.GetPods()
```

Comment:
After this change, pod will in activePods. When consumed by predicateAdmitHandler, I think this will cause an error.

Reply:
You are right, Thakns.